### PR TITLE
Embed libovsdb

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: pkg/cmd/server/server.go [hunk 1]
```diff
@@ -108,7 +108,11 @@ func main() {
 				srv := jrpc2.NewServer(assigner, servOptions)
 				handler.SetConnection(srv)
 				srv.Start(ch)
-				go func() { defer cancel(); srv.Wait() }()
+				go func() {
```
Comment (roytman) [inline pkg/cmd/server/server.go:111]: the changes are not related to the libovsdb

Comment (hunchback) [reply pkg/cmd/server/server.go]: yup, rebased (and removed this)
